### PR TITLE
Update Filter.php

### PR DESCRIPTION
**Pull request by mosiyash**

### CODE REVIEW

**Comment by harikt** [comment]
Could you please explain the problem here and probably add a test if it is failing ?

**Comment by mosiyash** [comment]
I'm not very at english, but...

Add test to FilterTest.php:

public function testAll()
{
    ...
    $this->filter->addMessages('zzz', 'First message');
        $this->filter->addMessages('zzz', 'Second message');
        $expect = [
            'First message',
            'Second message',
        ];
        $actual = $this->filter->getMessages('zzz');
        $this->assertSame($expect, $actual);
    }
}

## Test results:

nikita@nikita:~/dev/Aura.Input/tests$ phpunit
PHPUnit 4.2.2 by Sebastian Bergmann.

Configuration read from /home/nikita/dev/Aura.Input/tests/phpunit.xml

.................E......

Time: 78 ms, Memory: 4.75Mb

There was 1 error:

1) Aura\Input\FilterTest::testAll
Undefined index: zzz

/home/nikita/dev/Aura.Input/src/Aura/Input/Filter.php:133
/home/nikita/dev/Aura.Input/tests/Aura/Input/FilterTest.php:102

FAILURES!

## Tests: 24, Assertions: 53, Errors: 1.

2014-08-28 14:23 GMT+07:00 Hari K T notifications@github.com:

> Could you please explain the problem here and probably add a test if it is
> failing ?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/auraphp/Aura.Input/pull/43#issuecomment-53682014.

## 

C уважением, Никита Мосияш
nikita.mosiyash@gmail.com

File: tests/Aura/Input/FilterTest.php [hunk 1]
```diff
@@ -98,5 +98,15 @@ public function testAll()
...
+	$this->filter->addMessages('zzz', 'First message');
+	$this->filter->addMessages('zzz', 'Second message');
+	$expect = [
+		'First message',
+		'Second message',
+	];
+	$actual = $this->filter->getMessages('zzz');
+        $this->assertSame($expect, $actual);
     }
 }
```
Comment (harikt) [inline tests/Aura/Input/FilterTest.php:111]:
``` php
$expect = [
+ 'First message',
+ 'Second message',
+ ];
```

Is there a tab? If so please make to spaces.

**Comment by harikt** [comment]
no problem with the language. Just want to make sure what the fix was intended to.

Tests don't lie, so you can always send a test if you feel you are not so good to communicate.

Thanks once again for the PR and your efforts.

**Comment by pmjones** [comment]
Seeing as the test fails I can't merge the PR.

File: tests/Aura/Input/FilterTest.php [hunk 2]
```diff
@@ -98,5 +98,14 @@ public function testAll()
         $values->foo = 'foovalue';
         $passed = $this->filter->values($values);
         $this->assertTrue($passed);
+
+        // test undefined key
+        $this->filter->addMessages('zzz', 'First message');
+        $this->filter->addMessages('zzz', 'Second message');
+            'First message',
```
Comment (harikt) [inline tests/Aura/Input/FilterTest.php:105]: Hey you missed `$actual = [` .

**Pull request closed without merging**